### PR TITLE
Hotfix/refactor

### PR DESCRIPTION
**Pull request by gloriateodoro**
## Checklist
- [x]  Separa e refatora componentes Media e ImageLogo
- [x]  Modifica modo de exportação dos componentes utilizados na página inicial
- [x] Separa componente ImpactNumber
- [x] Renomeia componente (dentro de content) de Media para ContentMedia
- [x] Refatora Heading presente nas  incubations e about

### CODE REVIEW

**Comment by bnd-operator** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **lpirola**
You can assign the PR to them by writing `/assign @lpirola` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=nossas%2Fnossas-org).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/nossas/nossas-org/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by bnd-operator** [comment]
@gloriateodoro: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr | fef0a5bc929e0bfd481fdd000a26699a0dec6589 | [link](http://dashboard-jx.nossas.tech/namespaces/jx/pipelineruns/nossas-nossas-org-pr-55-pr-gqqrh) | `/test pr`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: components/Media/ImageLogo.tsx
Comment (lpirola) [inline components/Media/ImageLogo.tsx:10]: a variável isExternal não foi definida na função e não foi prevista como parametro.

File: pages/about/index.tsx
Comment (lpirola) [inline pages/about/index.tsx:48]: seria bom evitar ao máximo passar espaçamento em componentes específicos

**Comment by bnd-operator** [comment]
@gloriateodoro: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr | fef0a5bc929e0bfd481fdd000a26699a0dec6589 | [link](http://dashboard-jx.nossas.tech/namespaces/jx/pipelineruns/nossas-nossas-org-pr-55-pr-bw6qt) | `/test pr`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by bnd-operator** [comment]
:star: PR built and available in a preview **nossas-nossas-org-pr-55** [here](http://nossas-org-pr55.nossas.tech)

File: pages/about/index.tsx
Comment (lpirola) [inline pages/about/index.tsx:83]: muito bom manter apenas um componente!

File: pages/about/index.tsx
Comment (lpirola) [inline pages/about/index.tsx:192]: se eu entendi, o componente `Link` quando recebe essa propriedade como verdadeira, é adicionado o target _blank?

**Review by lpirola — APPROVED** [review]

**Comment by bnd-operator** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[lpirola](https://github.com/nossas/nossas-org/pull/55#pullrequestreview-688724974 "Approved")*

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=nossas%2Fnossas-org).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/nossas/nossas-org/blob/main/OWNERS)~~ [lpirola]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: pages/about/index.tsx
Comment (gloriateodoro) [reply pages/about/index.tsx:192]: @lpirola isso, Pira! é do próprio Chakra. Precisaria mesmo ser definido como parâmetro?

File: pages/about/index.tsx
Comment (gloriateodoro) [reply pages/about/index.tsx:48]: espaçamento = colocar as propriedades assim uma abaixo da outra?

**Pull request merged**
